tests/e2e/utils/authenticate-to-connect.ts

```diff
@@ -13,20 +13,25 @@ async function authenticateToConnect(): Promise<void> {
     expect(await text("Access my account").exists()).toBeTruthy();
     await click("Access my account");
 
-    expect(await text("Email").exists()).toBeTruthy();
-    await write(configVariables.connectTestAccountEmail);
-    await press("Enter");
-
-    expect(await text("Password").exists()).toBeTruthy();
-    await write(configVariables.connectTestAccountPassword);
-    await press("Enter", { navigationTimeout: 60000 });
-
-    const needScopeAcceptance = await text(
-      "would like to have access to these information about you",
-    ).exists(10000, 50);
-
-    if (needScopeAcceptance) {
-      await click("Accept");
+    const isAlreadyLoggedIn = await text("already logged in").exists(1000, 50);
+    if (isAlreadyLoggedIn) {
+      await click("Continue");
```

Comment:
When the user is already logged in there's the "Continue" button instead of the login form.
It happens on the Profile happy path since I separated profile and addresses tests (in the same file), so I added it in this function.

Reply:
We'll need to remember this when we'll refactor the auth middleware to redirect to `/account` from `/` when the user is already logged in.

Reply:
It could be great for consistency to update `fill-incorrect-identity-input-values.test.ts`, which is the only other test file that has two separated test cases, and make it use what you did with `profile-happy-path.test.ts` and `authenticate-to-connect.ts`

Reply:
Done 👍